packages/app-mobile/components/NoteEditor/MarkdownToolbar.tsx

```diff
@@ -0,0 +1,111 @@
+const { ScrollView, View, Text, TouchableHighlight } = require('react-native');
+const { useMemo } = require('react');
+import { themeStyle } from '@joplin/lib/theme';
+import { EditorControl, SelectionFormatting } from './EditorType';
+
+interface ToolbarProps {
+	editorControl: EditorControl;
+	selectionState: SelectionFormatting;
+	themeId: number;
+}
+
+interface ButtonProps {
+	onPress: ()=> void;
+	style?: any;
+	activated: boolean;
+	accessibilityLabel: string;
+	title: string;
+}
+
+
+const MarkdownToolbar = (props: ToolbarProps) => {
+	const styles = useMemo(() => {
+		const theme = themeStyle(props.themeId);
+
+		return StyleSheet.create({
+			button: {
+				padding: 20,
+				alignItems: 'center',
+				backgroundColor: theme.backgroundColor4,
+				color: theme.color4,
+			},
+			buttonActive: {
+				backgroundColor: theme.backgroundColor3,
+				color: theme.color3,
+			},
+			text: {
+				color: theme.color4,
+			},
+			toolbar: {
+				flex: 0,
+				flexDirection: 'row',
+				alignItems: 'baseline',
+				justifyContent: 'center',
+			},
+			toolbarContent: {
+				flexGrow: 1,
+				justifyContent: 'center',
+			},
+		});
+	}, [props.themeId]);
+
+	const ToolbarButton = (props: ButtonProps) => {
+		const activatedStyle = props.activated ? styles.buttonActive : {};
+
+		return (
+			<TouchableHighlight
+				style={{ ...styles.button, ...(props.style ?? {}), ...activatedStyle }}
+				onPress={props.onPress}
+				accessibilityLabel={props.accessibilityLabel}>
+				<Text style={{ ...styles.text, ...props.style }}>{props.title}</Text>
+			</TouchableHighlight>
+		);
+	};
+
+	return (
+		<View style={styles.toolbar}>
+			<ScrollView contentContainerStyle={styles.toolbarContent} horizontal={true}>
+				<ToolbarButton
+					onPress={props.editorControl.toggleBolded}
+					style={{ fontWeight: 'bold' }}
+					activated={props.selectionState.bolded}
+					title="B"
+					accessibilityLabel="Bold"/>
+				<ToolbarButton
+					onPress={props.editorControl.toggleItalicized}
+					style={{ fontStyle: 'italic' }}
+					activated={props.selectionState.italicized}
+					title="I"
+					accessibilityLabel="Italicize"/>
+				<ToolbarButton
+					onPress={props.editorControl.toggleInlineCode}
+					style={{ fontFamily: 'monospace' }}
+					activated={props.selectionState.inInlineCode}
+					title="``"
+					accessibilityLabel="Inline Code"/>
+				<ToolbarButton
+					onPress={props.editorControl.toggleList}
+					activated={props.selectionState.listLevel > 0}
+					title="•…"
+					accessibilityLabel="Toggle List"/>
+				<ToolbarButton
+					onPress={() => props.editorControl.toggleHeaderLevel(1)}
+					activated={props.selectionState.headerLevel == 1}
+					title="H1"
+					accessibilityLabel="Togle Header Level 1"/>
+				<ToolbarButton
+					onPress={() => props.editorControl.toggleHeaderLevel(2)}
+					activated={props.selectionState.headerLevel == 2}
+					title="H2"
+					accessibilityLabel="Toggle Header Level 2"/>
```

Comment:
These labels currently aren't localizable. How should this be handled in the mobile app?